projecteuler/problems/sums.py

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env python3
+
+def get_sum_of_multiples_of_three_or_five(maximum_number):
+    result = 0
+    for count in range(1, maximum_number):
+        if count % 3 == 0 or count % 5 == 0:
+            result += count
+    return result
+
+def get_sum_of_even_fibonacci_numbers(maximum_number):
+    current_number, last_number, next_number = 1, 1, 2
```

Comment:
can be done without the `next_number` variable

Reply:
I did think it must not need all three! Edited to use just current and last now.